packages/terra-notification-dialog/src/terra-dev-site/doc/example/NotificationDialogInModalManager.jsx

```diff
@@ -0,0 +1,83 @@
+import React, { useContext, useState } from 'react';
+import classNames from 'classnames/bind';
+
+import Button from 'terra-button';
+import { DisclosureManagerContext } from 'terra-disclosure-manager';
```

Comment:
Do we absolutely need this example? I hate pulling disclosure manager into this package.

Reply:
This example was added in https://github.com/cerner/terra-framework/pull/430 to fix the bug described as:
> Notification dialog doesn't account for additional focus trap sources within a modal manager 

It's probably a good idea to keep it 🤷‍♀️